generate

```diff
@@ -4,7 +4,7 @@
   cd _api-reference-generator/ &&
   rm -rf ../modules/ &&
   # Not optional, we need to be documenting the latest published release
-  npm update &&
+  # npm update &&
```

Comment:
Do we want to leave it this way? If we leave it this way then you are responsible for remembering to npm update, or to git pull in apostrophe if you're npm linked.

Reply:
I'll revert it. I meant to remove that.